agent-install/agent-uninstall.sh

```diff
@@ -317,10 +317,15 @@ function verifyNodeRemovedFromManagementHub() {
 
     set +e
     $KUBECTL exec ${POD_ID} -n ${AGENT_NAMESPACE} -c "anax" -- bash -c "${EXPORT_EX_USER_AUTH_CMD}; hzn exchange node list ${node_id}" >/dev/null 2>&1
-    if [ $? -ne 8 ]; then
-	    log_warning "Node was not removed from the management hub"
-    fi
+    local exit_code=$?
     set -e
```

Comment:
I do not think that this call is producing the error. based on the logs order it looks like it comes from line 303 in `deleteNodeFromManagementHub()`.